crates/autopilot/src/solvable_orders.rs

```diff
@@ -151,6 +154,7 @@ impl SolvableOrdersCache {
     /// the case in unit tests, then concurrent calls might overwrite each
     /// other's results.
     pub async fn update(&self, block: u64) -> Result<()> {
+        let start = Instant::now();
```

Comment:
Is this still needed if we have all the instrumentation inside the runloop?

Reply:
Don't we want to measure the time we spend on updating the solvable orders cache once a new block is received?

Reply:
My understanding is that this is put on the critical path of the runloop (cf. @MartinquaXD PR from yesterday), but ok if we want to be able to measure the update independent of that, let's keep this additional metric.